### PR TITLE
Feature/cloud-2359 python

### PR DESCRIPTION
**Pull request by jnesusx**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Added Docs 

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

```text
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
```

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -23,6 +23,16 @@ description: |
           ecr_repository: 'demo/example'
           python-version: 3.10
   ```
+
+  ## Locating Container Images
+
+  Python Docker images are located in AWS ECR, V-Prod. Their URI looks like the below snippet.
```
Comment (zumic96) [inline action.yml:29]: Can you make it same as for other actions.

File: action.yml [hunk 2]
```diff
@@ -23,6 +23,16 @@ description: |
...
+
+  ```yaml
+          064859874041.dkr.ecr.us-east-2.amazonaws.com/<octopusgroup>/<app name>
```
Comment (zumic96) [inline action.yml:32]: Remove this indent in code-block from all PRs.
No need for it to be specified as yaml, you can use plain code-block.

**Review by zumic96 — CHANGES_REQUESTED** [review]

**Review by vibindaniel — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (jnesusx) [reply action.yml]: changed to match others

File: action.yml (hunk 2 above)
Comment (jnesusx) [reply action.yml]: Changed yaml to text, removed indent

File: README.md [hunk 3]
```diff
@@ -26,6 +26,14 @@ permissions:
         ecr_repository: 'demo/example'
         python-version: 3.10
 ```
+
+## Locating Container Images
+
+Python Docker images are located in AWS ECR, V-Prod. Their URI looks like the below snippet.
```
[the remark this answers is not in the document]
Comment (jnesusx) [reply README.md]: changed to match others

**Review by vibindaniel — APPROVED** [review]

**Review by zumic96 — APPROVED** [review]

**Pull request merged**